llvm/lib/Target/RISCV/RISCVRegisterInfo.cpp

```diff
@@ -179,7 +179,8 @@ void RISCVRegisterInfo::adjustReg(MachineBasicBlock &MBB,
                                   const DebugLoc &DL, Register DestReg,
                                   Register SrcReg, StackOffset Offset,
                                   MachineInstr::MIFlag Flag,
-                                  MaybeAlign RequiredAlign) const {
+                                  MaybeAlign RequiredAlign,
+                                  bool IsPrologueOrEpilogue) const {
```

Comment:
Can we use the FrameSetup/FrameDestroy flag instead adding `IsPrologueOrEpilogue`?

Reply:
That's really good suggestion! applied :)